spyder/plugins/completion/providers/kite/widgets/calltoaction.py

```diff
@@ -94,6 +95,11 @@ def handle_mouse_press(self, event):
     def handle_processed_completions(self, completions):
         if not self.get_conf('kite_call_to_action'):
             return
+
+        installers_available = check_kite_installers_availability()
```

Comment:
@andfoy I was checking and maybe here is worthy to just use the config value (getting the response from the actual check and doing the http gave me a little bit of lag)?